WooCommerce new documentation (#1299)
mercadopago/devsite-docs: guides/plugins/woocommerce/sales-processing.en.md

# Sales Processing

> With all steps completed, your customers will be able to make purchases from your store. When performing a transaction, it is commom for some messages to return with specific information about the purchase. These informations are  available in **Notas do Pedido** section of the WooCommerce panel.

Reviewer: ```suggestion
With all steps completed, your customers will be able to make purchases from your store. When performing a transaction, it is common for some messages to return with specific information about the purchase. This information is available in the **Notas do Pedido** section of the WooCommerce panel.
```

![Sales processing](/images/woocomerce/en_sales_processing.gif)

In this section, you will have access to details of the possible status and reasons for each return.

## Payment Status

> Every sale generates a payment status that shows the status of the sale including confirmation, pending or denial of payment and other important information about the transaction. 

Reviewer: ```suggestion
Every sale generates a payment status that shows the condition of the purchase, including confirmation, pending, or denial of payment and other important information about the transaction.
```

> There are two statuses in a WooCommerce sale: _status do pedido_ and _status do pagamento_. In the table below you can check the details on how this information correlates.

Reviewer: ```suggestion
There are two statuses in a WooCommerce sale: _status do pedido_ and _status do pagamento_. In the following table, you can check the details on how this information correlates.
```

| Mercado Pago status (payment) | WooCommerce status (order) |
|---|---|
| Pending | Pending |
| Approved | Processing |
| Inprocess | On hold |
| Inmediation | On hold|
| Rejected | Failed |
| Cancelled | Cancelled |
| Refunded | Refunded |
| Chargedback| Refunded|

> For more information, access the [Activities](https://www.mercadopago.com.br/activities) section of your Mercado Pago account and the [WooCommerce order manual](https://docs.woocommerce.com/document/como-gerenciar-pedidos/).

Reviewer 2: ```suggestion
For more information, access the [Activities](https://www.mercadopago[FAKER][URL][DOMAIN]/activities) section of your Mercado Pago account and the [WooCommerce order manual](https://docs.woocommerce.com/document/como-gerenciar-pedidos/).
```

In addition, you can have access to the details of each payment attempt that happened in your store. To do this, go to the WordPress panel, and then click **WooCommerce > Orders > Order Details**.

## Reasons for refusals 

> Regarding the **approval of payments** in your store, there are three main reasons that can directly impact these results. 

Reviewer: ```suggestion
Regarding the **approval of payments** in your store, three main reasons can directly impact these results.  
```

Below, we detail the factors that influence a payment refusal.

| Reason | Situation | How to avoid |
|---|---|---|

> | Buyer's mistakes | Errors in filling in address, CPF or card details. | Checkout with clear information in step-by-step purchase. |

Reviewer: ```suggestion
| Buyer's mistakes | Errors in filling in address, CPF, or card details. | Checkout with clear information in the step-by-step purchase. |
```

> | Banking refusals | Cards with expiration date, lack of limit, insufficient funds or disabled for online purchases. | Offer alternatives to other methods and/or payment terms. |

Reviewer: ```suggestion
| Banking refusals | Cards with an expiration date, lack of limit, insufficient funds, or disabled for online purchases. | Offer alternatives to other methods and/or payment terms. |
```
